lib/blockscore/base.rb

```diff
@@ -53,22 +51,22 @@ def self.resource
       @resource ||= Util.to_underscore(to_s.split('::').last)
     end
 
-    def self.api_url
-      'https://api.blockscore.com/'
-    end
-
     def self.endpoint
       fail NotImplementedError, ABSTRACT_WARNING if equal?(Base)
 
-      "#{api_url}#{Util.to_plural(resource)}"
+      "#{API_URL}#{Util.to_plural(resource)}"
```

Comment:
If you follow the idea from https://github.com/BlockScore/blockscore-ruby/pull/67/files#diff-d64d05c8aeab202e4a33798a74a3eed0R8 you might be able to replace this with `API_URL.merge(path: Util.to_plural(resource)` avoiding string interpolation.